src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = function(num = 0) {
```

Comment:
```suggestion
  const adder = function(num) {
```
You just can check if undefined. See bellow...